DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/BlobHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GitHubExtention.Preferences.WebApi.Constants;
+using Microsoft.WindowsAzure;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    internal static class BlobHelper
+    {
+        public static CloudBlobContainer GetWebApiContainer()
+        {
+            string str =
+                CloudConfigurationManager.GetSetting(AvatarConstants.StorageConnectionString);
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(str);
+
+            CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
```

Comment:
None of the following components implement IDispoable (or a Close() method).

CloudStorageAccount
CloudBlobClient
CloudBlobContainer
CloudBlockBlob
those resources are released automatically by the storage client library, because they're created by the library. For example, using Reflector to review the code, you may notice:
executionState.RestCMD.ResponseStream.Dispose();
And:
executionState.Resp.Close();